docs/documentation/server_admin/topics/authentication/passkeys.adoc

```diff
@@ -11,3 +11,60 @@ Therefore, users of {project_name} can do passkey registration and authenticatio
 Both synced passkeys and device-bound passkeys can be used for both Same-Device and Cross-Device Authentication (CDA).
 However, passkeys operations success depends on the user's environment. Make sure which operations can succeed in https://passkeys.dev/device-support/[the environment].
 ====
+
+[[_passkeys-conditional-ui]]
+==== Passkey Authentication with Conditional UI
+
+Passkey Authentication with Conditional UI can authenticate a user with its passkey in the same way as in xref:_webauthn_loginless[LoginLess WebAuthn].
+This authentication shows a user a list of passkeys stored on a device where the user runs a browser.
+Therefore, the user can select one of the passkeys in the list to authenticate them. Compared with xref:_webauthn_loginless[LoginLess WebAuthn], the authentication improves the user's experience of authentication.
+
+[NOTE]
+====
+This authentication uses https://github.com/w3c/webauthn/wiki/Explainer:-WebAuthn-Conditional-UI/[WebAuthn Conditional UI].
```

Comment:
```suggestion
This authentication uses the https://github.com/w3c/webauthn/wiki/Explainer:-WebAuthn-Conditional-UI/[WebAuthn Conditional UI].
```

Reply:
Thank you. I will fix it as you suggested.